src/components/Tools.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+class Tools extends Component{
+    render(){
+        return (
+        
+            <div>
+                <h1>Tools page</h1>
+             <Card style={{ width: '18rem' }}>
+            <Card.Img variant="top" src="holder.js/100px180" />
+            <Card.Body>
+              <Card.Title>Content Collection</Card.Title>
+              <Card.Text>
+                All the contents of the student.
+              </Card.Text>
+              <Button variant="primary">Contents</Button>
+            </Card.Body>
+          </Card>
+          <Card style={{ width: '18rem' }}>
+            <Card.Img variant="top" src="holder.js/100px180" />
+            <Card.Body>
+              <Card.Title>Portfolios</Card.Title>
+              <Card.Text>
+                All the portfolios of the student.
```

Comment:
I mentioned it in pull request #68 but I'll ask again here because I didn't get any response: What do you mean by portfolios?  I'm just not sure what's supposed to go here and I want us to be careful not to add extra pages we won't have time to finish.

Reply:
I looked up blackboard and thought students should have portfolios i.e collections of their work, accomplishments, skills, potentials...etc.

Reply:
That sounds like something you might find on LinkedIn and could be useful, but I'm not sure it totally fits with a course management system.  But we could keep it for now and see if we have time to build off of it.